pkg/baur/inputresolver.go

```diff
@@ -202,6 +220,17 @@ func (i *InputResolver) pathsToUniqInputs(repositoryRoot string, paths []string)
 
 		dedupMap[path] = struct{}{}
 
+		excluded, err := i.globPathResolver.Matches(path, excludePatterns)
+		if err != nil {
+			return nil, fmt.Errorf("ExcludedFiles: %w", err)
+		}
+
+		if excluded {
+			// TODO: log the pattern that caused it's exclusion
```

Comment:
```suggestion
			// TODO: log the pattern that caused its exclusion
```

Reply:
Thanks for reminding me about that TODO comment. :-)
I implemented the missing part + removed the comment.

Reply:
glad i could be of any use :sweat_smile: